layout/templates/mobileMapUI.mustache

```diff
@@ -18,6 +18,6 @@
     </div>
   </div>
   <div class="zoom-out-button-container">
-    <button class="zoom-out-button">Zoom Out</button>
+    <button class="zoom-out-button hidden">Zoom Out</button>
```

Comment:
Is it too early to also work on making the state-level selector expand and shrink when the zoom out button is removed or added?

Reply:
I think the resizing bit will be the difficult thing

Reply:
I disagree, we just have to hide the div that the zoom-out button is in and flex-box will take over.

Reply:
shouldn't that be happening then? I use `display: none` for the zoom-out box but the dropdowns stay the same size. Is there a different way I should be hiding them?

Reply:
@lindsaycarr sorry was editing my post above, check it out now